chickadee/utils.py

```diff
@@ -0,0 +1,21 @@
+import logging
+from rpy2.robjects.packages import isinstalled, importr
+
+
+logger = logging.getLogger("PYWPS")
+logger.setLevel(logging.NOTSET)
+
+formatter = logging.Formatter(
+    "%(asctime)s %(levelname)s: osprey: %(message)s", "%Y-%m-%d %H:%M:%S"
+)
+handler = logging.StreamHandler()
+handler.setFormatter(formatter)
+logger.addHandler(handler)
+
+
+def get_ClimDown():
+    # Install and import R packages
+    if not isinstalled("ClimDown"):
+        utils = importr("utils")
+        utils.install_packages("ClimDown")
+    return importr("ClimDown")
```

Comment:
Is there some sort of error handling we could throw in here in case the package cannot be installed? Is there a way to pin the version we target?